app/models.py

```diff
@@ -535,6 +533,30 @@ def __init__(self,
         self.date_modified = date_modified
         self.release_date = release_date
 
+    def as_dict(self):
```

Comment:
This isn't being used anywhere but I left it in case it will come in handy later.